app/src/app/admin/articles/article/article.ts

```diff
@@ -70,7 +70,7 @@ namespace app.admin.articles.article {
                         return articleService.getArticle($stateParams.permalink)
                             .then((article) => {
 
-                                article._articleMetas = articleService.hydrateMetaFromTemplate(article._articleMetas, ArticleConfig.articleMetaTemplate);
+                                article._articleMetas = articleService.hydrateMetaFromTemplate(article, ArticleConfig.articleMetaTemplate);
```

Comment:
Why the change to passing in the full article object? It then requires that the function is aware of the structure of the article. I think the way you had it is better with pasing the articleMetas collection. Maybe it would make more sense to rename the function `hydrateMetaCollectionFromTemplate` ?